src/components/Form.js

```diff
@@ -157,15 +154,10 @@ export default class Form extends Component {
   };
 
   getRegistry() {
-    // For BC, accept passed SchemaField and TitleField props and pass them to
-    // the "fields" registry one.
-    const { fields, widgets } = getDefaultRegistry();
     return {
-      fields: { ...fields, ...this.props.fields },
-      widgets: { ...widgets, ...this.props.widgets },
-      ArrayFieldTemplate: this.props.ArrayFieldTemplate,
-      ObjectFieldTemplate: this.props.ObjectFieldTemplate,
-      FieldTemplate: this.props.FieldTemplate,
+      fields: this.props.fields,
+      widgets: this.props.widgets,
+      templates: this.props.templates,
```

Comment:
To maintain backwards compatibility with current props, you could check for the old props here and use them if they were provided.

Otherwise to be strict with [Semver](https://semver.org/) then including this package would mean including this PR would mean releasing a V2.

If we include backwards compatible props, then this could be release as V1.1.

For example (this is pseudocode), this could provide backwards compatibility
```
  templates: {
      ...this.props.templates,
      ArrayFieldTemplate: this.props.ArrayFieldTemplate || this.props.templates.ArrayFieldTemplate,
      ObjectFieldTemplate: this.props.ObjectFieldTemplate || this.props.templates.ObjectFieldTemplate,
      FieldTemplate: this.props.FieldTemplate || this.props.templates.FieldTemplate,
      ErrorListTemplate: this.props.ErrorList || this.props.templates.ErrorListTemplate
  }
```

@glasserc do you think that maintaining backwards compatibility with props is important?

Reply:
I thought that this would be in v2 but yes we can change it to support backward compatibility.